pkgs/by-name/fl/florist/package.nix

```diff
@@ -0,0 +1,36 @@
+{
+  stdenv,
+  gnat13,
+  gnat13Packages,
+  fetchFromGitHub,
+  lib,
+}:
+
+stdenv.mkDerivation rec {
+  name = "florist";
+  version = "24.2";
+  src = fetchFromGitHub {
+    owner = "adacore";
+    repo = "florist";
+    rev = "refs/heads/${version}";
```

Comment:
```suggestion
    rev = "refs/tags/${version}";
```

Reply:
The repository has only one tag v20.2, which refers to an older unstable version. Branches are maintained instead, and the latest stable is 24.2. That was the reason, why I have used "heads".